hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/model/DataEntity.java

```diff
@@ -86,6 +86,11 @@ public Blueprint(DataEntity.Role role, StructuredData value, Map<String, Object>
             this.value = value;
         }
 
+        // this is needed for Jackson deserialization
+        public Blueprint() {
```

Comment:
Jackson is fine with private constructors. IMHO this and the Update one should be private or at least protected so that users cannot instantiate this essentially invalid objects.